src/components/TodoFilterBar/TodoFilterBar.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import classNames from 'classnames';
+import { TodoContext } from '../../TodoContext';
+import { FilterField } from '../../types/FilterField';
+
+const TodoFilterBar = () => {
+  const {
+    todos,
+    filterField,
+    setFilterField,
+    clearCompleted,
+  } = React.useContext(TodoContext);
+
+  const uncompletedTodos = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="footer" data-cy="todosFilter">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${uncompletedTodos.length} items left`}
+      </span>
+
+      <ul className="filters">
+        <li>
```

Comment:
can you check some similar things in this link items, create some array, and iterate through it?
like
```
const VALUES = [{ path: '#/', filterValue: FilterField.all, label: 'All' }]


jsx
{VALUES.map(item => (
  <li key={item.path}>
    <a
      href={item.path}
      className={classNames({
        selected: filterField === item.filterValue,
      })}
      onClick={(event) => handleClick(event, item.filterValue)}
    >
      {item.label}
    </a>
  </li>
))}
```

Reply:
follow DRY principle (Don't Repeat Yourself)